src/main/java/org/folio/gobi/Mapper.java

```diff
@@ -177,23 +169,11 @@ private CompletableFuture<CompositePurchaseOrder> mapCompositePOLine(Map<Mapping
           contributors.add(contributor);
           pol.setContributors(contributors);
         });
-        setObjectIfPresent(reportingCode, o -> {
-          List<ReportingCode> reportingCodes = new ArrayList<>();
-          reportingCodes.add(reportingCode);
-          pol.setReportingCodes(reportingCodes);
-        });
-        setObjectIfPresent(claim, o -> {
-          List<Claim> claims = new ArrayList<>();
-          claims.add(claim);
-          pol.setClaims(claims);
-        });
-
```

Comment:
Do we also need to delete this line: setObjectIfPresent(license, o -> eresource.setLicense((License) o));
because we have deleted mapping for licenses ?

Reply:
yes , as we removed mapping so  "License" will always be empty object. i think its better to remove . Please review .